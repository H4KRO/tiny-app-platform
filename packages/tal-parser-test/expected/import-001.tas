let toto = import "toto"

toto.useExportedFunction()
