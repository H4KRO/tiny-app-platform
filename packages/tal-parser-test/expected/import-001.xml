<program>
  <declarelocal>
    <import />
  </declarelocal>
  <call>
    <attribute>
      <local />
    </attribute>
  </call>
</program>
